application/__tests__/example.macro.test.js

```diff
@@ -0,0 +1,168 @@
+/* globals describe test expect */
+
+const { JSDOM } = require('jsdom')
+const fs = require('fs')
+const path = require('path')
+const nunjucks = require('jstransformer')(require('jstransformer-nunjucks'))
+const { getDirectoryFromFilepath, isArray } = require('../filters/hmrc-design-system')
+const htmlEscape = htmlString => htmlString.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;')
+const nunjucksEscape = nunjucksString => nunjucksString.replace(/\{% raw %\}\n/, '').replace(/\{% endraw %\}/, '')
+const exampleId = 'test'
+
+const options = {
+  path: [
+    path.join(__dirname, '..', 'macros'),
+    path.join(__dirname, '..', 'partials'),
+    path.join(__dirname, '..', '..', 'src'),
+    path.join(__dirname, '..', '..', 'src'),
```

Comment:
Duplicate line, is this needed?

Reply:
good spot, fat fingered cut and paste, I'll remove.